src/VSSetup.PowerShell/VSSetup.PowerShell.csproj

```diff
@@ -41,10 +42,10 @@
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.PowerShell.3.ReferenceAssemblies.1.0.0\lib\net4\System.Management.Automation.dll</HintPath>
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\lib\PowerShell\System.Management.Automation.dll</HintPath>
```

Comment:
> Ick.

Yeah, but I could not think of any other way of providing that assembly, short of downloading and extracting the PS 2.0 SDK MSI as part of the build process.

> Perhaps I can get the asmmeta version

Would there be any chance of releasing it on NuGet, for the benefit of other projects targeting PS 2.0?